src/imp/openssl.rs

```diff
@@ -102,7 +102,9 @@ fn load_android_root_certs(connector: &mut SslContextBuilder) -> Result<(), Erro
             .filter_map(|e| fs::read(e.path()).ok())
             .filter_map(|b| X509::from_pem(&b).ok());
         for cert in certs {
-            connector.cert_store_mut().add_cert(cert)?;
+            if let Err(err) = connector.cert_store_mut().add_cert(cert) {
+                eprintln!("load_android_root_certs error: {:?}", err);
```

Comment:
Can we use `debug!` rather than `eprintln!` here?

Seems reasonable otherwise.

Reply:
Unfortunately rust-native-tls doesn't use crate log

```
error: cannot find macro `debug!` in this scope                                                                       
   --> /tmp/rust-native-tls/src/imp/openssl.rs:106:17                                                                 
    |                                                                                                                 
106 |                 debug!("load_android_root_certs error: {:?}", err);                                             
    |                 ^^^^^                                                                                           
```

Should I add it  ?

Reply:
Yep, we can just add the dependency.